docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       - rabbitmq_data:/var/lib/rabbitmq
 
   django:
-    image: geonode/django
+    image: geonode/geonode
```

Comment:
It's "informative", for newcomers it indicates that this is the main applicative container.